openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -103,6 +103,18 @@
             });
 
             sectionOneFieldViews = [
+                new FieldsView.DateFieldView({
```

Comment:
@marcotuts could you let us know if you'd like us to move the joined date to the bottom again? Thanks!